scripts/make_release.py

```diff
@@ -15,43 +14,31 @@
 END_HEADER_MAGIC = b"@@END_HEADER@@"
 
 
-def get_artifacts_zip(artifacts, artifact_name):
-    r = requests.get(
-        artifacts[artifact_name]["archive_download_url"],
-        headers={"Authorization": f"token {token}"},
-        stream=True
-    )
-    r.raise_for_status()
-    decoding_classes = {
-        "application/zip": zipfile.ZipFile
-    }
-    return decoding_classes[r.headers['Content-Type']](BytesIO(r.content))
-
-
-def get_version(version, installer_metadata):
-    if version is None:
-        # If no version is given, convert the build version number to a release version
-        version = Version(installer_metadata["VER"])
-        version = ".".join(map(str, version.release))
-    version = Version(version)
-
-    if version.is_prerelease:
-        # Bump the pre-release digit
-        next_version = [
-            ".".join(map(str, version.release)),
-            "".join(map(str, version.pre[:-1] + (version.pre[-1] + 1,))),
-        ]
-    else:
-        # Bump the least significant digit
-        next_version = [
-            ".".join(map(str, version.release[:-1] + (version.release[-1] + 1,))),
-            "a1"
-        ]
-
-    return str(version), "".join(next_version)
-
-
 def main(run_id=None, requested_version=None, workflow_fn="build-and-test.yml"):
+    """Make a new release of DIRACOS2 based on a GitHub Actions CI run
+
+    Releases are made by:
+    1. Download the artefact from a GitHub Actions CI run
```

Comment:
```suggestion
    1. Download the artifact from a GitHub Actions CI run
```
Cf. `get_artifacts_zip` below. I know the "e" is the British spelling, but consistency.